packages/augur-ui/src/modules/app/actions/init-augur.ts

```diff
@@ -214,10 +217,39 @@ export function connectAugur(
       preloadAccount(ACCOUNT_TYPES.TORUS);
     }
 
-    let provider;
-    if (windowApp.web3) {
-      provider = new Web3Provider(windowApp.web3.currentProvider);
+    let provider = null;
+    let networkId = null;
+
+    if (process.env.ETHEREUM_NETWORK === 'kovan') {
```

Comment:
Can this be moved out to helper. Maybe with helper to convert networkId to network name